tools/setup-windows/setup-windows.csproj

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\Configuration.props" />
```

Comment:
I only looked at the pull request for 455202000ee9beefc25e07eabe52ca910bd495dd to see if there was a discussion about an important reason to have `Configuration.props` imported later, and I didn't see anything there. But there might have been earlier discussion in another PR about a reason it should be imported at the end of the file. So feel free to close this PR and take a different approach if needed. Thanks!